apostrophecms/a3-docs
# add new GA4 tag and remove unsupported tag manager

## Summary

*Summarize the changes briefly, including which issue/ticket this resolves. If it closes an existing Github issue, include "Closes #[issue number]"*
This PR adds the new GA4 tag to the documentation site and removes the old tag. It also removes the plugin for adding a Google analytic tag to the site, as it does not support the new GA4 tags. Closes OS-134.

## What are the specific steps to test this change?

*For example:*
> 1. Run the website and log in as an admin
> 2. Open a piece manager modal and select several pieces
> 3. Click the "Archive" button on the top left of the manager and confirm that it should proceed
> 4. Check that all pieces have been archived properly

1. Build the documentation.
2. Open the DevTools and look for the *second* Google Analytic tags manager script. It should have a value of "G-t!M7W6BWMD"

## What kind of change does this PR introduce?
*(Check at least one)*


## Make sure the PR fulfills these requirements:


If adding a new feature without an already open issue, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**
